src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -47,6 +89,21 @@ public static JsonDocument Parse(ReadOnlySequence<byte> utf8Json, JsonReaderOpti
             }
         }
 
+        /// <summary>
+        ///   Parse a <see cref="Stream"/> as UTF-8-encoded data representing a single JSON value into a
+        ///   JsonDocument.  The Stream will be read to completion.
+        /// </summary>
+        /// <param name="utf8Json">JSON data to parse.</param>
+        /// <param name="readerOptions">Options to control the reader behavior during parsing.</param>
+        /// <returns>
+        ///   A JsonDocument representation of the JSON value.
```

Comment:
📝 There are a bunch more cases where `<see cref` is missing around type references.

Reply:
They're following what I _think_ the style guide is, which is the first usage in a given section is a link and the others are not.  And I didn't see=thistype, since that felt pretty weird.